backend/FwLite/LcmCrdt/CrdtProjectsService.cs

```diff
@@ -145,7 +146,7 @@ private void EnsureDeleteProject(string sqliteFile)
                 await Task.Delay(1000);
                 try
                 {
-                    File.Delete(sqliteFile);
+                    await db.EnsureDeletedAsync();
```

Comment:
funny so it was actually like this previously, but I had issues with that as well so I made it retry and use File.Delete instead. One issue with this approach is that `EnsureDeleteProject` is a triggered task which will keep running and the caller doesn't wait for it to finish (by design), the issue is that when `CreateProject` returns it will dispose of the service scope which will dispose of the DbContext, so I'm not sure if this will blow up in a different way. Maybe we should also try `FileDelete`? if `EnsureDeleted` throws?

At the very least we should have at least 1 test which ensures we delete on a throw, should be easy to test since we can provide an AfterCreate action from testing

Reply:
We decided that we'll call `EnsureDeleteAsync` before we kick of this task, if that throws **then** we'll kick of this task as before.
And we'll add a test that throws in `AfterCreate`